src/libPersistence/BlockStorage.cpp

```diff
@@ -588,7 +684,7 @@ bool BlockStorage::PutDSCommittee(const shared_ptr<DequeOfNode>& dsCommittee,
                                   const uint16_t& consensusLeaderID) {
   LOG_MARKER();
 
-  lock_guard<mutex> g(m_mutexDsCommittee);
+  unique_lock<shared_timed_mutex> g(m_mutexDsCommittee);
   m_dsCommitteeDB->ResetDB();
```

Comment:
Can call `BlockStorage::ResetDB(DBTYPE)` instead

Reply:
Will put in next pr